### ISSUE TITLE
Scrolling to last index in list (bottom) render problem

### ISSUE THREAD

**Issue by hazzo** [hilongjw/vue-recyclerview#8]
Hi @hilongjw, excelente method for scrolling to index but when you set the last index of list the recyclerview renders **to many tombstones** in dom.

Thanks again!

Edited: It doesn't render items it renders to many tombstones

**Comment by hilongjw** [comment]
@hazzo, Thank you for your feedback, scrollToIndex (index) by calling fill (0, index) to achieve, there will be such a problem, I will try to modify here, make it automatically recycle unnecessary tombstones.

**Comment by hazzo** [comment]
My pleasure, i'm looking forward to use this for production in a chat app 😄 

Okay, you are forcing the scroll by filling the recyclerview with the elements before the index to scroll.

Maybe it could be fixed with a fetch from last index to top? What I mean, instead of fetching from the beginning of the array list start from the bottom, and instead of pushing elements unshift them.

Just an idea.

**Comment by hilongjw** [comment]
Wow, sounds good, 👍. My thoughts are similar to you:

```javascript
 scrollToIndex (index) {
    const lastItemCount = this.lastAttachedItem_ - this.firstAttachedItem_
    this.fill(index - lastItemCount, index + 1)
  },
```

**Comment by hilongjw** [comment]
fixed at 0.3.1

**Comment by hazzo** [comment]
Perfect! Now DOM it's cleaner!
